src/main/java/school/faang/bjs2_80710_stream_1/ListOperations.java

```
@@ -0,0 +1,82 @@
+package school.faang.bjs2_80710_stream_1;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public class ListOperations {
+
+    public static int sumOfEvenNumbers(List<Integer> numbers) {
+        return numbers.stream().reduce(0, (sum, number) ->
+                number % 2 == 0 ? sum + number : sum
+        );
+    }
+
+    public static Optional<Integer> findMax(List<Integer> numbers) {
+        return numbers.stream().max(Comparator.comparingInt(a -> a));
+    }
+
+    public static Optional<Double> findAverage(List<Integer> numbers) {
+        if (numbers == null || numbers.isEmpty()) {
```

В целом тут можно и опустить проверки на null, задача всё же о работе со stream api )
С другой стороны проверки на нулл никогда лишними не бывают
С третьей стороны, лично мне в большинстве случаев более близок fail fast подход, и лучше выбрасывать исключения, если что-то невалидное пришло в метод (напр. @NonNull ломбока)
По Optional так же, моё мнение, что в этих задачах лучше было бы завершать с .orElseThrow(...)

Но повторюсь - задача о работе со stream api и твое решение так же корректно. Поэтому не воспринимай как замечание, просто мысли вслух )

---

Я почему-то решил что ломбок только для конструкторов @nonnull предлагает. Добавил в методы, дышать легче стало. 
.orElseThrow(...) как я понимаю можно вызвать только на Optional объекте, в этом задании пожалуй тут и где максимум - единственные места. Но взял на заметку, в будущем пригодится!
Отличные мысли, очень за них признателен, продолжай делиться!